plugins/star-rating/api/api.js

```diff
@@ -260,6 +261,31 @@ const widgetPropertyPreprocessors = {
             }
         });
     };
+    var nonChecksumHandler = function(ob) {
+        var params = {
+            no_checksum: true,
+            //providing data in request object
+            'req': {
+                url: "/i?" + ob.params.href.split("/i/feedback/input?")[1]
+            },
+            //adding custom processing for API responses
+            'APICallback': function(err, responseData, headers, returnCode, paramsOb) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['paramsOb' is defined but never used.](https://app.codacy.com/manual/ar2rsawseen/countly-server/pullRequest?prid=5592913)